src/stringobj.c

```diff
@@ -529,8 +530,11 @@ static Obj CopyString(Obj list, Int mut)
 /****************************************************************************
 **
 *F  PrintString(<list>) . . . . . . . . . . . . . . . . . . .  print a string
+*F  FuncCHUNK_ESCAPE_STRING(<list>) . . . . . . . escape a string as a string
 **
 **  'PrintString' prints the string with the handle <list>.
+**  'CHUNK_ESCAPE_STRING' returns a list of strings which, when concatenated,
+**  produce a String containing what PrintString outputs.
```

Comment:
What I don't understand: Why not instead provide a function `VIEW_STRING_FOR_STRING` which returns the already concatenated result? Do you have any applications in mind where one would need these separate "chunks"?

Reply:
Nope, just generally trying to make the C-level code as simple as possible